modules/gui/src/com/haulmont/cuba/gui/components/filter/FilterDelegateImpl.java

```diff
@@ -1821,7 +1822,24 @@ protected void applyFts() {
 
             if ((applyTo != null) && ListComponent.class.isAssignableFrom(applyTo.getClass())) {
                 if (clientConfig.getGenericFilterFtsTableTooltipsEnabled()) {
-                    filterHelper.initTableFtsTooltips((ListComponent) applyTo, adapter.getMetaClass(), searchTerm);
+                    filterHelper.initTableFtsTooltips(
+                            (ListComponent) applyTo,
+                            (listComponent, entityId) -> {
+                                if (listComponent instanceof Table) {
+                                    CollectionDatasource<Entity<Object>, Object> ds = listComponent.getDatasource();
+                                    TableItems tableItems = ((Table)listComponent).getItems();
```

Comment:
Do not use listComponent.getDatasource. Use ListComponent#getItems() and inheritors `DatasourceDataUnit, ContainerDataUnit` for metaClass extracting.